src/main/java/edu/tamu/iiif/service/AbstractManifestService.java

```diff
@@ -165,10 +165,22 @@ protected Model getRdfModel(String url) throws NotFoundException {
     }
 
     protected String getRdf(String url) throws NotFoundException {
-        Optional<String> rdf = Optional.ofNullable(restTemplate.getForObject(url, String.class));
-        if (rdf.isPresent()) {
-            logger.debug("RDF for {}: \n{}\n", url, rdf.get());
-            return rdf.get();
+        try {
+            if (logger.isDebugEnabled()) {
+                logger.info("Requesting RDF for {}", url);
+            }
+            Optional<String> rdf = Optional.ofNullable(restTemplate.getForObject(url, String.class));
+            if (rdf.isPresent()) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug("RDF for {}: \n{}\n", url, rdf.get());
+                }
+                return rdf.get();
+            }
+        } catch (RestClientException e) {
+            logger.error("Failed to get RDF for {}: {}", url, e.getMessage());
+            if (logger.isDebugEnabled()) {
```

Comment:
Same situation regarding `logger.isDebugEnabled()` as the comment about on line 174.